src/main/java/dev/emi/trinkets/api/client/TrinketRenderer.java

```diff
@@ -9,7 +9,7 @@
 import net.minecraft.util.math.Vec3f;
 
 public interface TrinketRenderer {
-	public static final float MAGIC_ROTATION = 180f / (float) Math.PI;
+	float MAGIC_ROTATION = 180f / (float) Math.PI;
```

Comment:
It feels weird to not have this annotated with `final`, especially as a capitalized field.

Reply:
All members (both fields and methods) of an interface are implicitly public, which explains why I removed `public` from all the interface methods.

Additionally, all fields of an interface are always static and final. See [this stackoverflow post](https://stackoverflow.com/questions/1513520/why-are-all-fields-in-an-interface-implicitly-static-and-final) for a good explanation.

Reply:
I understood that this was implicit from the change but I just meant it looks weird to have allcaps without final, probably because I'm not used to it

Reply:
Ah I can see why. From looking around a bit though this seems pretty standard.